### PR TITLE
Add Matt Reynolds to the authors file.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/site/_data/authorsData.json [hunk 1]
```diff
@@ -1377,5 +1377,14 @@
...
+  },
+  "mattreynolds": {
+    "name": {
+      "given": "Matt",
+      "family": "Reynolds"
+    },
+    "description": {
+      "en": "Matt is a software developer"
+    },
+  },
```
Comment (jpmedley) [inline src/site/_data/authorsData.json:1389]:
```suggestion
  }
```

**Comment by jpmedley** [comment]
@robdodson From the test:
>Error: authors [mattreynolds] have no posts and/or Twitter information

There aren't any posts by him because the first one is still in a PR. How do I get past this test?

**Comment by robdodson** [comment]
@jpmedley what about adding matt to the authors file in the post PR you mentioned?

**Comment by kaycebasques** [comment]
@jpmedley please follow Rob's suggestion. Add the author data to the PR where they first become a web.dev author.

**Pull request closed without merging**